tectonic/xetex-core.h

```diff
@@ -87,6 +87,10 @@ typedef struct XeTeXLayoutEngine_rec* XeTeXLayoutEngine;
 #define FONT_FLAGS_COLORED  0x01
 #define FONT_FLAGS_VERTICAL 0x02
 
+/* Windows? */
+// This one is in xetex-code/source/texk/web2c/xetexdir/xetex.h, but not here? Baffled as to why not.
+#define M_PI 3.14159265358979323846
```

Comment:
It seems that for Windows you need to \#define _USE_MATH_DEFINES` before including `math.h`. So you do not need to redefine it yourself.

See https://msdn.microsoft.com/en-us/library/4hwaceh6.aspx